benchmarks/src/test/java/org/apache/druid/benchmark/query/CachingClusteredClientBenchmark.java

```diff
@@ -343,7 +344,7 @@ public <T, QueryType extends Query<T>> QueryToolChest<T, QueryType> getToolChest
         processingConfig,
         forkJoinPool,
         QueryStackTests.DEFAULT_NOOP_SCHEDULER,
-        new MapJoinableFactory(ImmutableSet.of(), ImmutableMap.of()),
+        new JoinableFactoryWrapper(new MapJoinableFactory(ImmutableSet.of(), ImmutableMap.of())),
```

Comment:
fixed, now all tests use a unified `new JoinableFactoryWrapper(NoopJoinableFactory.INSTANCE)` from `JoinableFactoryWrapperTest.NOOP_JOINABLE_FACTORY_WRAPPER` 
the moving average query tests don't use it right now since that'll require adding test-jar dep from processing in moving average, which doesn't seem necessary at this point.